QualityControl/lib/repositories/LayoutRepository.js

```diff
@@ -24,16 +24,31 @@ export class LayoutRepository extends BaseRepository {
    * @param {number} [options.owner_id] - Filter layouts by owner ID
    * @param {string} [options.name] - Filter layouts by exact name match
    * @param {Array<string>} [options.fields] - Array of field names to include in each returned layout object
+   * @param {string} [options.filter] - Filter layouts by containing filter.objectPath, case insensitive
```

Comment:
I believe following the changes in the previous PR, the filter now is an object rather than a string?